### ISSUE TITLE
error in powerpack-commit

### ISSUE THREAD

**Issue by M1cha** [papyros/powerpack#5]
sudo ./powerpack/powerpack-commit papyros

==> Installing packages to /home/m1cha/powerpack/build/papyros/pacstrap
:: Synchronizing package databases...
error: failed retrieving file 'papyros.db' from disk : Couldn't open file //home/m1cha/powerpack/packages/papyros.db
error: failed to update papyros (download library error)
 core                                                          120.9 KiB  1259K/s 00:00 [##################################################] 100%
 extra                                                        1805.2 KiB  2006K/s 00:01 [##################################################] 100%
 community                                                       2.5 MiB  1112K/s 00:02 [##################################################] 100%
error: target not found: ostree-git
warning: skipping target: linux
error: target not found: ostree-git
error: target not found: xdg-app-git
error: target not found: qml-extras
error: target not found: qml-material
error: target not found: papyros-shell
error: target not found: qt5-base-git
error: target not found: qt5-declarative-git
error: target not found: qt5-wayland-dev-git
==> ERROR: Failed to install packages to new root
##

**Comment by crutchcorn** [comment]
I have the same issue. I think it is related to Issue #6

**Comment by githubgeeker** [comment]
you have to install that packages first...

**Comment by crutchcorn** [comment]
Do you mean that you have to install ostree first? Isn't powerpack-build supposed to install the needed software for this? Or are there dependancies that are expected to be installed first?

**Comment by githubgeeker** [comment]
yes you should install in your host machine the necessary packages. I don't
remember what, I have to look for them in my archlinux
Il 05/feb/2015 15:19 "crutchcorn" notifications@github.com ha scritto:

**Comment by iBelieve** [comment]
@crutchcorn You'll need OSTree (and as a dependency, linux-user-chroot) installed from the AUR before  running `powerpack-commit`, as it needs OSTree to compose the repository.

@M1cha Did you run `powerpack-build` first? You need to build the extra packages before composing the OSTree repository.

**Comment by crutchcorn** [comment]
@iBeliever Just did so. Upon running powerpack-build again I get issue #6 still.

**Comment by M1cha** [comment]
@iBeliever yes I did, but I didn't install the qt5 packages yet, I'll try again when I did.
